test/return64bit/testbind.js

```diff
@@ -4,14 +4,18 @@
 // returned via the accessor method getTempRet0()
 
 Module['runtest'] = function() {
-  var low = _test_return64(0x11223344, 0xaabbccdd);
-  var high = getTempRet0();
+  // Use eval to create BigInt, as no support for Xn notation yet in JS
+  // optimizer.
```

Comment:
Hmm.. how can that be true?  We use the `Xn` notation all over the place and run the generated code through the JS optimizer

Reply:
Oh I see this code is just moving around... I guess we can update it after.

Reply:
That diff is a little deceptive. What I actually did is rename testbind.js to testbind_nobigint.js and testbind_bigint.js to testbind.js. So that is just the pre-existing diff between those files.